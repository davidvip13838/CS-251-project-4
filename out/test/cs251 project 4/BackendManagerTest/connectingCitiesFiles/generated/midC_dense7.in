500
3 9
40 5
28 23
7 44
19 39
10 37
37 0
30 35
6 27
20 7
28 2
13 49
33 12
2 47
15 28
0 34
7 20
43 46
28 43
37 48
21 49
34 36
34 44
41 13
25 17
46 25
5 20
2 26
27 45
35 27
42 19
0 16
8 32
41 2
22 23
8 29
0 37
23 12
11 20
35 32
15 45
16 14
33 33
23 14
47 25
40 22
22 42
39 0
31 36
20 50
44 5
11 17
21 45
3 6
20 29
26 25
25 25
49 28
43 23
10 28
46 17
26 20
34 6
22 16
32 20
0 28
5 43
40 14
32 2
18 0
29 9
38 19
6 7
21 34
17 19
50 45
32 36
32 22
26 12
27 46
18 4
22 33
22 4
36 47
35 0
10 42
6 13
39 36
23 4
14 33
19 31
43 2
0 17
10 4
9 14
29 24
41 10
30 39
19 42
4 30
23 7
13 46
38 15
41 16
35 26
25 34
20 1
47 27
45 10
17 34
25 32
18 3
15 3
47 23
34 41
44 6
3 36
37 44
31 33
33 32
42 27
1 23
47 40
28 25
20 2
7 26
14 17
10 0
22 47
22 2
40 9
16 50
41 19
16 5
37 17
11 3
4 37
38 0
19 32
17 7
22 46
39 14
36 50
2 25
15 32
7 15
6 33
10 47
34 48
12 25
18 15
17 39
31 28
13 21
30 19
18 6
26 16
38 48
47 18
15 35
35 43
2 17
29 41
41 6
41 27
19 33
10 31
31 26
10 48
43 17
7 42
42 46
30 14
48 35
1 7
40 26
7 17
1 38
46 28
43 39
39 19
21 46
39 6
13 43
26 7
34 10
32 21
21 36
14 30
37 50
14 48
14 41
45 8
20 38
26 22
27 32
24 14
33 11
4 9
16 33
7 37
41 44
33 1
9 9
24 42
8 38
6 10
28 47
5 15
44 32
12 29
12 9
43 26
13 45
48 14
40 24
22 21
39 12
27 23
36 16
7 6
45 3
34 5
39 8
30 42
37 35
6 22
42 30
11 41
6 6
1 9
39 49
4 3
40 42
1 43
39 29
19 34
25 8
5 36
47 35
14 8
22 44
4 21
28 21
6 39
13 17
25 30
13 14
8 2
28 1
32 14
21 6
3 3
45 22
50 0
44 7
33 4
23 44
29 7
42 1
8 49
39 31
9 31
36 42
47 33
16 32
25 9
40 44
34 29
28 3
40 25
24 24
35 9
48 20
41 45
7 41
6 28
29 12
32 35
40 37
25 14
43 15
31 42
40 41
45 12
4 28
49 38
0 25
42 26
48 1
42 16
26 45
37 30
2 50
1 36
2 33
28 26
37 29
1 4
13 6
15 48
31 34
13 24
11 4
5 40
37 16
24 16
27 3
34 28
15 39
46 37
8 19
5 13
7 3
40 38
3 34
23 26
4 6
42 49
5 6
40 15
50 33
16 3
40 1
43 33
2 4
33 48
23 34
40 31
1 34
0 7
18 8
40 50
15 42
28 22
22 45
32 41
31 12
17 36
42 29
39 16
34 9
36 19
27 34
48 6
43 6
8 11
42 20
7 48
35 45
13 1
0 14
49 1
38 10
45 18
32 49
7 32
32 31
19 6
46 16
50 18
31 50
30 30
8 25
47 30
11 37
31 48
0 39
0 5
18 14
4 10
47 15
3 41
23 21
20 23
24 5
4 39
37 7
11 50
36 36
6 3
48 5
45 47
49 42
34 45
44 42
48 43
20 15
50 28
32 48
24 45
49 15
1 47
1 35
17 10
18 18
6 47
20 21
37 21
4 27
24 23
36 11
19 24
40 11
37 27
15 38
8 4
38 20
6 17
16 12
3 27
3 10
3 40
19 50
10 1
28 42
18 19
43 37
25 11
5 37
3 42
8 36
28 24
3 25
15 24
9 38
9 46
5 21
38 6
44 4
28 16
33 24
27 39
47 16
13 3
2 10
34 50
1 31
13 41
25 41
46 12
19 38
2 44
14 2
7 13
42 37
44 1
49 48
18 22
11 24
5 48
25 7
37 9
1 27
14 21
36 1
24 10
0 8
20 35
9 21
29 1
11 15
47 2
41 17
18 1
47 50
40 48
10 33
44 28
42 5
45 49
45 2
9 49
5 8
12 18
32 18
26 18
26 48
25 33
38 31
29 27
22 27
26 13
3 5
10 25
43 11
6 31
8 14
40 17
32 6
17 48
9 47
48 19
23 35
3 45
4 4
28 13
21 40
4 45
9 16
