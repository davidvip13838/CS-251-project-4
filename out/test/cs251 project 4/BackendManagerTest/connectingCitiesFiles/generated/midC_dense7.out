785.7815123821778